doc/api/querystring.markdown

```diff
@@ -29,7 +29,7 @@ Example:
     // Suppose gbkEncodeURIComponent function already exists,
     // it can encode string with `gbk` encoding
     querystring.stringify({ w: '中文', foo: 'bar' }, null, null,
-      { encodeURIComponent: gbkEncodeURIComponent })
+      { encodeURIComponent: 'gbkEncodeURIComponent' })
```

Comment:
I think this is intentional. The comment above says that this is supposed to be a function that already exists.